app/controllers/api/reservations_controller.rb

```diff
@@ -4,7 +4,14 @@ def index
     render json: @reservations
   end
 
-  def create; end
+  def create
+    @reservation = Reservation.new(reservation_params)
+    if @reservation.save
+      render json: @reservation, status: :created, location: api_reservation_url(@reservation)
+    else
+      render json: @reservation.errors, status: :unprocessable_entity
+    end
+  end
 
   private
 
```

Comment:
please add user ID to the params. I get this error when I try to make a reservation

{
    "user": [
        "must exist"
    ]
}

For the post to work well we need the user's ID

Reply:
Oh yeah. That is also my mistake. I've added some changes to make it work and depend on the user, as all calls to `GET` should include the current user's ID. Since we haven't added Devise yet, I suggest you pull my changes and try with `http://localhost:3000/api/reservations?user_id=1`